src/evict/evict_lru.c

```diff
@@ -2932,14 +2933,14 @@ __wti_evict_app_assist_worker(WT_SESSION_IMPL *session, bool busy, bool readonly
         WT_STAT_CONN_INCR(session, application_cache_ops);
         WT_STAT_CONN_INCRV(session, application_cache_time, elapsed);
         WT_STAT_SESSION_INCRV(session, cache_time, elapsed);
-        if (busy) {
-            WT_STAT_CONN_INCR(session, application_cache_busy_ops);
-            WT_STAT_CONN_INCRV(session, application_cache_busy_time, elapsed);
-            WT_STAT_SESSION_INCRV(session, cache_time_busy, elapsed);
+        if (!interruptible) {
+            WT_STAT_CONN_INCR(session, application_cache_mandatory_ops);
+            WT_STAT_CONN_INCRV(session, application_cache_mandatory_time, elapsed);
+            WT_STAT_SESSION_INCRV(session, cache_time_mandatory, elapsed);
```

Comment:
If I understand correctly, `application_cache_ops == application_cache_mandatory_ops + application_cache_interruptible_ops`

So perhaps we could leave out `application_cache_mandatory_ops`. A user or engineer can always derive it from the other two stats. 

If you want to keep it, I suggest calling this category `application_cache_uninterruptible_ops` (etc) to make it more clear how it relates to the other stats